src/news_app/news_app.js

```diff
@@ -0,0 +1,52 @@
+import { makeFeed } from './create_feed';
+import { getData } from './get_data';
+import { clearFeed } from './clear_feed';
+// import { updateFeed } from './update_feed.js';
+
+import './news_style.css';
```

Comment:
do we need this here ?

Reply:
Styles? I was trying to keep this modules style out of the main css - which can become long very quick

Reply:
I can transfer them over if required